course_access_groups/serializers.py

```diff
@@ -1,13 +1,75 @@
-from rest_framework import serializers
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, unicode_literals
 
+from rest_framework import serializers
+from openedx.core.lib.api.serializers import CourseKeyField
 from course_access_groups.models import (
-    CourseAccessGroup,
+    Group,
+    CourseGroup,
+    Membership,
+    Rule,
 )
 
 
 class GroupSerializer(serializers.ModelSerializer):
+    id = serializers.ReadOnlyField()
+    organization_name = serializers.ReadOnlyField(source='organization.name')
+
+    class Meta:
+        model = Group
+        fields = [
+            'id', 'name', 'description', 'organization', 'organization_name',
+        ]
+
+
+class MembershipSerializer(serializers.ModelSerializer):
+    id = serializers.ReadOnlyField()
+    user_email = serializers.ReadOnlyField(source='user.email')
```

Comment:
do we need `user_` to preface `email`? What other email would it be?

Reply:
Same as username. I'd like to be explicit.

Reply:
Same reply as for username :)